### PR TITLE
migrating storage broker to clowder

### PR DESCRIPTION
**Pull request by gburges**

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+APP_NAME="storage-broker"  # name of app-sre "application" folder this component lives in
```
Comment (bsquizz) [inline pr_check.sh:6]: Needs to be 'ingress'

File: pr_check.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+COMPONENT_NAME="storage-broker"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/insightrs-storage-broker"
```
Comment (bsquizz) [inline pr_check.sh:8]: Typo here

File: pr_check.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+APP_NAME="storage-broker"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="storage-broker"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/insightrs-storage-broker"  
+
+IQE_PLUGINS=""
```
Comment (bsquizz) [inline pr_check.sh:10]: Set to "ingress"

**Comment by gburges** [comment]
(test)

**Comment by gburges** [comment]
[test]

**Comment by gburges** [comment]
[test]

**Comment by bsquizz** [comment]
[test]

**Pull request closed without merging**